site/en/docs/devtools/javascript/source-maps/index.md

```diff
@@ -1,98 +1,83 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Map Preprocessed Code to Source Code"
+title: "Map preprocessed code to source code"
 authors:
   - megginkearney
   - pbakaus
+  - sofiayem
 date: 2015-04-13
-#updated: YYYY-MM-DD
-description: "Keep your client-side code readable and debuggable even after you've combined, minified or compiled it."
+updated: 2023-03-29
+description: "Keep your client-side code readable and debuggable even after you've combined, minified, or compiled it."
 ---
 
 Keep your client-side code readable and debuggable even after you've combined, minified or compiled
 it. Use source maps to map your source code to your compiled code.
 
-## Summary {: #summary }
+## Get started with preprocessors {: #get_started_with_preprocessors }
 
-- Use Source Maps to map minified code to source code. You can then read and debug compiled code in
-  its original source.
-- Only use preprocessors capable of producing Source Maps.
-- Verify that your web server can serve Source Maps.
+To read and debug the original source of compiled code in the **Sources** panel, use source maps to map minified code to source code. 
 
-## Get started with preprocessors {: #get_started_with_preprocessors }
+To use source maps:
 
-This article explains how to interact with JavaScript Source Maps in the DevTools Sources Panel.
+- Use only the preprocessors that can produce source maps.
+- Verify that your web server can serve source maps.
 
 ## Use a supported preprocessor {: #use_a_supported_preprocessor }
 
-You need to use a minifier that's capable of creating source maps.
-For an extended view, see the [Source maps: languages,
-tools and other info][4] wiki page.
-
-The following types of preprocessors are commonly used in combination with Source Maps:
+The following types of preprocessors are commonly used in combination with source maps:
 
 - Transpilers ([Babel][5], [Traceur][6])
 - Compilers ([Closure Compiler][7], [TypeScript][8], [CoffeeScript][9], [Dart][10])
```

Comment:
```suggestion
- Compilers ([TypeScript][8], [Dart][10])
```
Closure Compiler is not maintain soon as well, remove it, Coffees cript is old so remove as well. Remove the reference URLs, Check if Dart URL updated.